client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -261,202 +227,150 @@ const RecurringPaymentsPlanAddEditModal = ( {
-			'Each amount you add will create a separate payment plan. You can create many of them.'
-		);
-		return (
-			<>
-				<p>{ product && product.ID ? editProduct : noProduct }</p>
-				<FormFieldset>
-					<FormLabel htmlFor="currency">{ translate( 'Select price' ) }</FormLabel>
-					{ product && (
-						<Notice
-							text={ translate(
-								'Updating the price will not affect existing subscribers, who will pay what they were originally charged on renewal.'
-							) }
-							showDismiss={ false }
-						/>
-					) }
-					<FormCurrencyInput
-						name="currency"
-						id="currency"
-						value={ currentPrice }
-						onChange={ handlePriceChange }
-						currencySymbolPrefix={ currentCurrency }
-						onCurrencyChange={ handleCurrencyChange }
-						currencyList={ currencyList }
-						placeholder="0.00"
-					/>
-					{ ! isFormValid( 'price' ) && (
-						<FormInputValidation
-							isError
-							text={ translate( 'Please enter a price higher than %s', {
-								args: [
-									formatCurrency(
-										minimumCurrencyTransactionAmount(
-											currentCurrency,
-											connectedAccountDefaultCurrency
-										),
-										currentCurrency
-									),
-								],
-							} ) }
-						/>
-					) }
-				</FormFieldset>
-				<FormFieldset>
-					<FormLabel htmlFor="renewal_schedule">
-						{ translate( 'Select renewal frequency' ) }
-					</FormLabel>
-					<FormSelect id="renewal_schedule" value={ editedSchedule } onChange={ onSelectSchedule }>
-						<option value="1 month">{ translate( 'Renew monthly' ) }</option>
-						<option value="1 year">{ translate( 'Renew yearly' ) }</option>
-						<option value="one-time">{ translate( 'One time sale' ) }</option>
-					</FormSelect>
-				</FormFieldset>
+	const addPlan = editedPostsEmail
+		? translate( 'Set up newsletter payment options' )
+		: translate( 'Set up payment options' );
+
+	const editPlan = editedPostsEmail
+		? translate( 'Edit newsletter payment options' )
+		: translate( 'Edit payment options' );
+
+	const editing = product && product.ID;
+	return (
+		<Dialog
+			isVisible={ true }
+			onClose={ onClose }
+			buttons={ [
+				{
+					label: translate( 'Cancel' ),
+					action: 'cancel',
+				},
+				{
+					label: translate( 'Save' ),
+					action: 'submit',
+					disabled: ! isFormValid(),
+					isPrimary: true,
+				},
+			] }
+		>
+			<FormSectionHeading>{ editing ? editPlan : addPlan }</FormSectionHeading>
+			<div className="memberships__dialog-sections">
 				<FormFieldset>
-					<FormLabel htmlFor="title">
-						{ translate( 'Please describe your payment plan' ) }
-					</FormLabel>
+					<FormLabel htmlFor="title">{ translate( 'Describe the offer' ) }</FormLabel>
 					<FormTextInput
 						id="title"
 						value={ editedProductName }
 						onChange={ onNameChange }
 						onBlur={ () => setFocusedName( true ) }
 					/>
+					<FormSettingExplanation>
+						{ translate( "Let your subscriber know what they'll get when they subscribe." ) }
```

Comment:
We got some feedback that the first `subscriber` is not exactly correct, since they're not a subscriber yet. Would it make sense to replace it with a `reader` or an `audience`?

Reply:
Hi @dlind1!
`Audience` seems to be more accurate in this case, `reader` sounds too generic.

Thank you!

Reply:
```suggestion
						{ translate( "Let your audience know what they'll get when they subscribe." ) }
```